lib/teiserver/lobby/libs/lobby_restrictions.ex

```diff
@@ -0,0 +1,293 @@
+defmodule Teiserver.Lobby.LobbyRestrictions do
+  @moduledoc """
+  Helper methods for lobby policies
+  """
+  alias Teiserver.CacheUser
+  require Logger
+  alias Teiserver.Battle.BalanceLib
+  alias Teiserver.Battle
+
+  @rank_upper_bound 1000
+  @rating_upper_bound 1000
+  @splitter "---------------------------"
+  @spec rank_upper_bound() :: number
+  def rank_upper_bound, do: @rank_upper_bound
+  def rating_upper_bound, do: @rating_upper_bound
+
+  def get_lobby_restrictions_welcome_text(state) do
+    play_level_bounds = get_rating_bounds_text(state)
+    play_rank_bounds = get_rank_bounds_text(state)
+
+    cond do
+      play_level_bounds == nil && play_rank_bounds == nil ->
+        []
+
+      true ->
+        ["This lobby has the following play restrictions:", play_level_bounds, play_rank_bounds]
+        |> Enum.filter(fn x -> x != nil end)
+    end
+  end
+
+  def get_rating_bounds_text(state) do
+    get_rating_bounds_for_title(state)
+  end
+
+  def get_rank_bounds_text(state) do
+    get_rank_bounds_for_title(state)
+  end
```

Comment:
What is the purpose of this? Why not call `get_rank_bounds_for_title(state)` or get_rating_bounds_for_title(state)` directly?

Reply:
Updated